### ISSUE TITLE
The Athlete project uses the LifeCycle CDM as well as its own CDM and would like both CDMs displayed in its variable list - Waiting for Parisa to discuss with Martine what to do with 'near' duplicates Athlete/LifeCycle.

### ISSUE THREAD

**Issue by EleanorHyde-UMCG** [molgenis/molgenis-emx2#2438]
**Is your feature request related to a problem? Please describe.**
The Athlete project makes use not only of its own CDM but also the LifeCycle CDM. When viewing the 'Athlete variables' you only get to see the variables in the Athlete_CDM and not the LifeCycle_CDM.

**Describe the solution you'd like**
The network data manager should be able to indicate for his/her project which CDMs should be visible when viewing that project's variable. This filter should then be automatically applied when viewing that project's list of variables.

**Describe alternatives you've considered**
The alternative is to go and look at the LifeCycle variables, but a researcher unfamiliar with the link with LifeCycle will miss these. 

**Additional context**
This feature would be useful for more projects than just Athlete.

EDIT:
Additional comment from Martine Vrijheid during IHEN meeting in November 24: she finds it difficult that with for example  'ADHD' variables in Athlete, you don't see that one of them is actually from LifeCycle. 

Is this something we can help?

Additional info from Brenda: 
We can show LifeCycle variables under ATHLETE, but the problem is that some LifeCycle variables have ben replicated in the ATHLETE CDM, so we cannot just show all LifeCycle variables. Someone from AHLETE needs to pick which to show.

Additional info from Parisa:

EU CHILD NETWORK 
https://data-catalogue.molgeniscloud.org/catalogue/ssr-catalogue/EUChildNetwork 
- I guess this would be the ideal place to find all LIFECYCLE + ATHLETE variables together. 
- If I click one of the variables listed, SBP_ for example it would be nice to see which network (ATHLETE or other) this variable was harmonized under. 
- Also here the networks (ATHLETE, LIFECYCLE, LONGITOOLS, ETC…) appear as cohorts under the harmonizations. They shouldn't as they are not cohorts and do not report variables.

**Comment by BrendaHijmans** [comment]
We can use the table Reused variables to pick those variables that they want to show in ATHLETE. Someone from ATHLETE should tell us which vaiables from LifeCycle they are using.

**Comment by EleanorHyde-UMCG** [comment]
> We can use the table Reused variables to pick those variables that they want to show in ATHLETE. Someone from ATHLETE should tell us which vaiables from LifeCycle they are using.

OK great. Do we just literally need a list of the ATHLETE variables with a cross in a column 'copied from LifeCycle' where appropriate, or is it not that simple? If you can define exactly how you'd like the information to be sent to us, I will ask the ATHLETE data manager to give us it.

**Comment by BrendaHijmans** [comment]
Indeed, someone can just delete all the variables that are not needed in ATHLETE from this complete list of LifeCycle CDM variables: [LifeCycle_variables.xlsx](https://github.com/user-attachments/files/18536648/LifeCycle_variables.xlsx). We would be able to upload the remaining variables to the Reused variables table.

**Comment by EleanorHyde-UMCG** [comment]
Great, will ask the data manager to do that. Thanks.

**Comment by EleanorHyde-UMCG** [comment]
Waiting for Parisa to come back with list of reused variables.

**Comment by EleanorHyde-UMCG** [comment]
19.2.25 sent Parisa a reminder

**Comment by EleanorHyde-UMCG** [comment]
19.2.25 Email from Parisa; I went over with Martine the showing of the catalogues, and it´s fine if we don’t include LIFECYCLE under ATHLETE and rather they both appear under the EUCCN with the other birth cohort projects. This makes it easier for researchers looking for birth cohorts. There shouldn´t be duplicates between the two as ATHLETE is an extension of LIFECYCLE so variables were chosen that were not included in LIFECYCLE.
